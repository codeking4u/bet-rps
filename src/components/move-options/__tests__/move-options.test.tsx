import renderer from "react-test-renderer";

import MoveOptions from "../move-options.component";

test("matches snapshot", () => {
  const tree = renderer.create(<MoveOptions />).toJSON();
  expect(tree).toMatchSnapshot();
});
