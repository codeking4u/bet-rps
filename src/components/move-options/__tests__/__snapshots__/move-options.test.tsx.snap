// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="betPosition"
>
  <div>
    PICK YOUR POSITIONS
  </div>
  <div
    className="betContainer "
  >
    <div
      className="betOption rock  "
      onClick={[Function]}
    >
      <div>
        Rock
         
      </div>
    </div>
    <div
      className="betOption paper  "
      onClick={[Function]}
    >
      <div>
        Paper
         
      </div>
    </div>
    <div
      className="betOption scissors  "
      onClick={[Function]}
    >
      <div>
        Scissors
         
      </div>
    </div>
  </div>
</div>
`;
